### DISCUSSION TITLE
A new module for the ags bar

### DISCUSSION THREAD

**Discussion by RipDr19** [end-4/dots-hyprland#219] [Ideas]
Can be it done so, that when there is no media playing, that space can display some other information like sysinfo (ram, cpu, disk etc) or maybe weather updates? Like sort of a stacked widget kind of setup, personally I have no experience with ags so, idk how or if this will be possible or not

**Comment by end-4** [comment]
That does make sense since that space is a little empty. 
Though for desktop users there's already ram/swap indicator as a fallback for battery display. what do you think would be suitable there, if we were to move those indicators to the left side?

**Reply by RipDr19** [reply]
Maybe weather updates, since not only will that bring more dynamicity and additional function but will be useful too, using wttr.py scripts I guess, also stacking isn't possible right?(I mean the music module only getting activated when some music plays?)

**Reply by end-4** [reply]
it is possible. Since I consider ram/swap display to be more important than weather, I'll have those shown on the music side and have weather as a battery fallback

**Reply by RipDr19** [reply]
Btw what do you mean by battery fallback, and while we are at it, can you add cpu usage too?

**Reply by end-4** [reply]
> what do you mean by battery fallback

Desktop = no battery. Battery indicator would just show as a red indicator with value = -1, which looks ugly

> can you add cpu usage too

Sure, that would make it look even less empty

**Reply by RipDr19** [reply]
Brilliant man will look forward to it
